openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -661,14 +641,6 @@ def _delete_cookie(request, response):
         httponly=settings.SESSION_COOKIE_HTTPONLY or None,
     )
 
-    # Log the cookie, but cap the length and base64 encode to make sure nothing
```

Comment:
This is just the old code that we don’t see a use for?

Reply:
Yeah. We never use this information, despite it logging all the time, and depending on _why_ there's a malformed cookie I could see there being sensitive information leaking from other cookies or even other headers (in a malformed request.) So I think we should just drop it.

(I assume it's usually logging due to expired cookies.)